chain/jsonrpc/src/lib.rs

```diff
@@ -51,6 +51,25 @@ mod metrics;
 const JSON_PAYLOAD_MAX_SIZE: usize = 2 * 1024 * 1024;
 const QUERY_DATA_MAX_SIZE: usize = 10 * 1024;
 
+/// RPCs that should only be called when the node is synced
+const RPC_REQUIRES_SYNC: [&'static str; 15] = [
```

Comment:
Should we inverse the logic and have a list of methods that don't require synced state? This way we lower the risk of missing the method in the future when we refactor, for example.

Reply:
I think that is less optimal because if a method is not in that list, you still have to check whether it is valid. If you do not check its validity we might risk doing a query for nothing.